cpp/include/ucxx/worker.h

```diff
@@ -643,6 +674,56 @@ class Worker : public Component {
   std::shared_ptr<Listener> createListener(uint16_t port,
                                            ucp_listener_conn_callback_t callback,
                                            void* callbackArgs);
+
+  /**
+   * @brief Register allocator for active messages.
+   *
+   * Register a new allocator for active messages. By default, only one allocator is defined
+   * for host memory (`UCS_MEMORY_TYPE_HOST`), and is used as a fallback when an allocator
+   * for the source's memory type is unavailable. In many circumstances relying exclusively
+   * on the host allocator is undesirable, for example when transferring CUDA buffers the
+   * destination is always going to be a host buffer and prevent the use of transports such
+   * as NVLink or InfiniBand+GPUDirectRDMA. For that reason it's important that the user
+   * defines those allocators that are important for the application.
+   *
+   * If the `memoryType` has already been registered, the previous allocator will be
+   * replaced by the new one. Be careful when doing this after transfers have started, there
+   * are no guarantees that inflight messages have not already been allocated with the old
+   * allocator for that type.
+   *
+   * @code{.cpp}
+   * // context is `std::shared_ptr<ucxx::Context>`
+   * auto worker = context->createWorker(false);
+   *
+   * worker->registerAmAllocator(`UCS_MEMORY_TYPE_CUDA`, ucxx::RMMBuffer);
+   * @endcode
+   *
+   * @param[in] memoryType  the memory type the allocator will be used for.
+   * @param[in] allocator   the allocator callbable that will be used to allocate new
```

Comment:
```suggestion
   * @param[in] allocator   the allocator callable that will be used to allocate new
```

Reply:
Fixed in https://github.com/rapidsai/ucxx/pull/48/commits/bbf6e27c014efea57e6cb0484d3b663dbb1e258e